.php_cs

```diff
@@ -17,18 +17,181 @@ $finder = PhpCsFixer\Finder::create()
     ->in(__DIR__ . '/tests')
     ->in(__DIR__ . '/tests/curated/files/*')
     ->in(__DIR__ . '/parsers')
-;
+    ->exclude('vendor')
+    ->append([__FILE__]);
 
 return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
-        '@PSR2' => true,
-        'binary_operator_spaces' => ['align_double_arrow' => true, 'align_equals' => true],
-        'single_quote' => false,
-        'array_syntax' => ['syntax' => 'short'],
-        'concat_space' => ['spacing' => 'one'],
-        'dir_constant' => true,
+        '@PSR2'                                       => true,
+        '@Symfony'                                    => true,
+        '@Symfony:risky'                              => true,
+        '@PHP70Migration'                             => true,
+        '@PHP71Migration'                             => true,
+        '@PHPUnit60Migration:risky'                   => true,
```

Comment:
Please remove the redundant rules defined below that are included in these rule sets (I've marked each line that's included in one of these rule sets if the option's value defined here matches the default of the rule).